_includes/sidebar-releases.json

```diff
@@ -1,70 +1,54 @@
+{% assign today = "today" | date: "%Y-%m-%d" %}
+{% assign v_prod = site.data.versions | where_exp: "v_prod", "v_prod.asst_supp_exp_date > today" | where_exp: "v_prod", "v_prod.release_date < today" | sort: "release_date" | reverse | map: "major_version" %}
+{% assign v_test = site.data.versions | where_exp: "v_test", "v_test.release_date > today or v_test.release_date == 'N/A'" | sort: "release_date" | last | map: "major_version" %}
 {
   "title": "CockroachDB Releases",
   "items": [
+    {
+      "title": "All Releases",
+      "urls": [
+        "/releases/index.html"
+      ]
+    },
     {
       "title": "Production Releases",
       "items": [
-        {
-          "title": "Latest v21.2",
-          "urls": [
-            "/releases/v21.2.5.html"
-          ]
-        },
-        {
-          "title": "Latest v21.1",
-          "urls": [
-            "/releases/v21.1.13.html"
-          ]
-        },
-        {
-          "title": "Latest v20.2",
-          "urls": [
-            "/releases/v20.2.19.html"
-          ]
-        },
+        {% for x in v_prod %}
+          {% assign r_latest = site.data.releases | where_exp: "r_latest", "r_latest.major_version == x" | where: "withdrawn", "false" | sort: "release_date" | last | map: "version" %}
+          {
+            "title": "Latest {{ x }}",
+            "urls": [
+              "/releases/{{ r_latest }}.html"
+            ]
+          },
+        {% endfor %}
         {
           "title": "All Production Releases",
```

Comment:
This currently sends you to https://deploy-preview-12877--cockroachdb-docs.netlify.app/docs/releases/v21.2.0.html. I'm not even sure we need this anymore. `All Releases` gets you the same thing. So this section could probably just have the latest support prod versions. 

Actually, thinking through this more, what if we refactor this sidenav menu a bit:

```
Releases
   CockroachDB Releases
       All Releases
       Latest 21.2
       Latest 21.1
       Latest 20.2
       Latest Testing Release
       Release Support Policy
       ...
```

Reply:
Done.